### ISSUE TITLE
Cannot find module.

### ISSUE THREAD

**Issue by amaciver** [skonves/express-http-context#19]
`npm install --save express-http-router`
`var httpContext = require('express-http-context');`
`npm start`
`cannot find module`

when I run `node` console, I can require the module fine. 
The folder in in my `node_modules` for the project.

Any ideas?

**Comment by dalgleish** [comment]
Are you are using webpack or some bundler that would interfere with how modules are loaded?

**Comment by amaciver** [comment]
no, I am not; using `nodemon` only. The strange thing is that some new
packages that I load in are `requiring` fine, but this one is not. It's a
puzzle.
